docs/Installation/Installation_Guide.rst

```diff
@@ -500,7 +481,7 @@ parameters as appropriate.)
       -  zenko.local
 
     cloudserver:
-     endpoint: "zenko.local"
+       endpoint: "zenko.local"
 

```

Comment:
The guide I've been following to install Zenko is here https://github.com/scality/Zenko/tree/development/0.9/charts

It gives the option of installing Zenko using flags in a copy-paste command, instead of editing `options.yml`

I feel like it would be good to at least link this guide somewhere (if not include it fully) so others can find it, since the guide there is very complete and discusses other flags/values for initial installation as well as upgrading + validating your deployment.

Reply:
Today's discourse has suggested a profusion of alternate installation methods. My intention with this guide is to put out something out that is simple and proven, that we can support. I don't dismiss your suggestion out of hand, nor irrevocably, but I don't think this failing to provide alternative/advanced techniques should be a blocker. Can this slide, or do you feel strongly otherwise?

Reply:
Could you add something at the end like,

For more installation options and info, please see *link to the other guide*.

If not for this PR, then one in the close future so we can start consolidating docs (or at least chaining them together so finding one let's you know of the others)